app/views/setup/worker_bootstrap.html.slim

```diff
@@ -1,15 +1,23 @@
-h1 Initial Doing Whatever Exactly This Step Does 2
-p Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
-h2 This is a Subtitle Just in Case We Need One
-p Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
-p The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.
+/
+  .alert.alert-info.alert-dismissible role="alert"
+    button.close[type="button" data-dismiss="alert" aria-label="Close"]
+      span[aria-hidden="true"]
+        | &times;
+    i.fa.fa-eye.fa-4x.pull-left aria-hidden="true"
+    span Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+
+h1 Worker setup
+p In order to bootstrap a kubernetes cluster in the next step, you will need to start some workers on which kubernetes is going to be deployed. You will need at least 2 workers for a minimal deployment. The specifications of the machines you will use, depend mostly on the applications which you plan to run on top of your cluster but we suggest at least 512Mb of RAM per Worker (#TODO: change this to what we need). Your Workers will need to be able to communicate with this controller node so make sure your network setup allows that (#TODO: mention specific ports?). 
+
+
+h2 Booting a worker
```

Comment:
@eotchi Looks awesome! I will only change `PATH_TO_PROFILE` to `URL_TO_PROFILE` to avoid any confusion about wether this can be a remote URL or not. I will also fix a typo `doing the bootstrap` -> `during the bootstrap`. I will keep the url example as you sent it until we implement the autoyast profile.